idea/testData/inspectionsLocal/foldInitializerAndIfToElvis/MultiLineInitializer.kt

```diff
@@ -0,0 +1,14 @@
+fun foo(vararg args: String): String? = null
+
+fun test(): Int {
```

Comment:
It seems that here we have wrong test data with redundant tab's.
This test does not green.